site/_collections/feeds.js

```diff
@@ -23,21 +27,34 @@
 module.exports = collection => {
   /** @type FeedsCollection */
   const feeds = {};
-  const tags = ['devtools'];
+  const tags = Object.keys(supportedTags);
```

Comment:
I'm not sure if we need an RSS feed for every supported tag... Practically speaking I doubt most folks would subscribe to them.

I think I prefer the idea of having the following feeds:

- blog (this would be the default feed for the site)
- all (this would be blog + all other docs)
- devtools (this would be specific to devtools)

In the future we may add others that are exclusive to tags. So perhaps it would be worth having a yml that defines the tags that we turn into RSS feeds.